graphql/schema/wrappers.go

```diff
@@ -673,7 +673,15 @@ func (f *field) DgraphAlias() string {
 	// if this field is repeated, then it should be aliased using its dgraph predicate which will be
 	// unique across repeated fields
 	if f.op.inSchema.repeatedFieldNames[f.Name()] {
-		return f.DgraphPredicate()
+		dgraphAlias := f.DgraphPredicate()
```

Comment:
This could have been simplified by appending the if below to the if above in this block with an and condition.